server/repositories/TreeRepository.js

```diff
@@ -31,7 +31,7 @@ class TreeRepository extends BaseRepository {
       WHERE
         t1.id IS NOT NULL
         AND	t2.id = :id
-        AND (t2.tree_id IS NULL OR t2.tree_id <> t1.id )
+        OR t2.tree_id IS NULL
```

Comment:
This should be reverted, this query gets trees that are within a certain distance of a capture, if the capture does not have a valid tree_id value or if it has a valid tree_id, the tree_id does not correspond to the id of the tree it is joined with